css/styles.css

```diff
@@ -0,0 +1,3 @@
+
+
+/*# sourceMappingURL=styles.css.map */
```

Comment:
You seem to be running Grunt to create a `styles.css` file, but you're not writing any SASS. You need to either use the SASS tools or remove them from your Gruntfile. I highly recommend you use SASS to get used to writing it and compiling it to CSS